openedx_learning/core/publishing/models.py

```diff
@@ -13,36 +24,258 @@
 
 
 class LearningPackage(models.Model):
+    """
+    Top level container for a grouping of authored content.
+
+    Each PublishableEntity belongs to exactly one LearningPackage.
+    """
+
```

Comment:
I had the same question. I guess it's about namespacing identifiers? 

I have the vague impression that the `publishing` app is not as generic as it could be when it's tied to the concept of LearningPackage. But maybe I'm just not clear on the role of LearningPackage.